pyserini/2cr/msmarco-v1-passage.yaml

```diff
@@ -495,10 +539,56 @@ conditions:
           - MAP: 0.4021
             nDCG@10: 0.6523
             R@1K: 0.7861
+  - name: splade-pp-ed-otf-onnx
+    display: "SPLADE++ CoCondenser-EnsembleDistil with ONNX Runtime"
+    display-html: "SPLADE++ CoCondenser-EnsembleDistil with ONNX Runtime"
+    display-row: "[2]"
+    command: python -m pyserini.search.lucene --threads 16 --batch-size 128 --index msmarco-v1-passage-splade-pp-ed --topics $topics --onnx-encoder SpladePlusPlusEnsembleDistil --output $output --hits 1000 --impact
+    topics:
+      - topic_key: msmarco-passage-dev-subset
+        eval_key: msmarco-passage-dev-subset
+        scores:
+          - MRR@10: 0.3830
```

Comment:
hey @ArthurChen189 why don't the numbers match those from Anserini? e.g., https://github.com/castorini/anserini/blob/master/docs/regressions-msmarco-passage-splade-pp-ed-onnx.md

I spot checked a few, and they seemed to be the pre-encoded queries?

Reply:
I've re-run experiments and put checkpoint Pyserini did call Anserini's encoder functions and with the ONNX model. Results are still the same as the ones in this pr.